docs/_guide/contracts-in-action/call-params.md

```diff
@@ -0,0 +1,22 @@
+# Call parameters
+
+Call parameters are:
+
+1. Amount;
+2. Asset ID;
+3. Gas forwarded.
+
+You can use these to forward coins to a contract. You can configure these parameters by creating an instance of `CallParams` and passing it to a chain method called `callParams`.
+
+At a basic level we can invoke the `callParams` method as part of the transaction flow to set these params:
+[@code:typescript](./packages/fuel-gauge/src/call-test-contract.test.ts#typedoc:Contract-call-params)
+
+Here we set call params alongside transaction params:
```

Comment:
Suggestion: this section (`Call parameters`) comes before the section `Tx Params`, but mentions transactions params; users might not know what it means. Can we have `Tx Params` (which could be renamed to `Transaction parameters` IMO) before `Call parameters`?